### PR TITLE
15153 customize styling browseable rest api

### PR DESCRIPTION
**Pull request by arthanson**
### Fixes: #15153 

Limits on what can be changed without copying and modifying the base template into api.html, but made tweaks to bring it more in-line with app.  Just kept inline css as fairly minor changes needed and IMHO not worth tooling for separate css file.

Monosnap Circuits | NetBox REST API 2024-04-01 09-35-12

### CODE REVIEW

File: netbox/templates/rest_framework/api.html [hunk 1]
```diff
@@ -2,6 +2,33 @@
 {% load static %}
 {% load i18n %}
 
+{% block bootstrap_theme %}
+  <link rel="stylesheet" type="text/css" href="{% static "rest_framework/css/bootstrap.min.css" %}"/>
+  <style>
```
Comment (jeremystretch) [inline netbox/templates/rest_framework/api.html:7]: We shouldn't have inline CSS anywhere. Any custom styles should be defined as SASS under `project-static/styles/` along with the native stylesheets. We can build and package a custom CSS just for the REST web UI.

**Review by jeremystretch — CHANGES_REQUESTED** [review]
It's a good start, but I see a few issues:

1. The navbar is not aligned properly
2. No visual contrast between the navbar and page content
3. Too much white space

rest_web_ui

**Comment by arthanson** [comment]
Changed to dark header and adjusted spacing / alignment of items

**Comment by jeremystretch** [comment]
We're going to punt on this one for now and revisit once we've spent more time improving the overall UI design.

**Pull request closed without merging**